Classes/KIFAccessibilityEnabler.m

```diff
@@ -38,8 +42,39 @@ + (instancetype)sharedAccessibilityEnabler
     return _sharedAccessibilityEnabler;
 }
 
+- (void)setApplicationAccessibilityEnabled:(BOOL)enabled
+{
+    // This works as of iOS 9.
+    CFPreferencesSetAppValue((CFStringRef)@"ApplicationAccessibilityEnabled",
+                             kCFBooleanTrue, (CFStringRef)@"com.apple.Accessibility");
+    CFPreferencesSynchronize((CFStringRef)@"com.apple.Accessibility",
+                             kCFPreferencesCurrentUser, kCFPreferencesAnyHost);
+    CFNotificationCenterPostNotification(CFNotificationCenterGetDarwinNotifyCenter(),
+                                         (CFStringRef)@"com.apple.accessibility.cache.app.ax",
+                                         nil, nil, enabled);
```

Comment:
The last argument for CFNotificationCenterPostNotification is "deliverImmediately":

"If true, the notification is delivered to all observers immediately, even if some observers are in suspended (background) applications and they requested different suspension behavior when registering for the notification. If false, each observer’s requested suspension behavior is respected.

If center is a Darwin notification center, this value is ignored."

While the value is ignored, and thus should not affect execution, it would be best to just set it to TRUE (instead of YES, since it is type Boolean and not BOOL, or variable enabled, since there is no change in behavior either way) to make the code easier to read.